.github/workflows/ci.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
         run: poetry install --sync
 
       - name: Run tests
-        run: poetry run pytest -m "not agentjobs and not integration" --cov-report xml
+        run: poetry run pytest -m "not agentjobs and not integration" --cov-report xml -ra
```

Comment:
This adds a summary report that shows what tests (if any) had a status other than `pass`.
https://docs.pytest.org/en/latest/how-to/output.html#producing-a-detailed-summary-report